src/main/java/edu/kis/vh/nursery/list/Node.java

```diff
@@ -2,11 +2,35 @@
 
 public class Node {
 
-    public int value;
-    public Node prev, next;
+    protected int value;
```

Comment:
All fields should be private after encapsulation.

value is immutable - should be final (setter unnecessary)